test/sagemaker_tests/pytorch/inference/integration/sagemaker/test_mnist.py

```diff
@@ -24,7 +24,7 @@
 
 
 @pytest.mark.model("mnist")
-@pytest.mark.multinode("multinode")
+@pytest.mark.multinode("multinode(needs_clarification)")
```

Comment:
@ChuyangDeng could you help us provide clarification on how many nodes this test will run on? I see that it's marked as initial_instance_count=1, but unsure if it's deterministic how many nodes are exercised here

Reply:
@metrizable could you comment as well?

Reply:
synced up with @ajaykarpur -- will remove multinode marker since it's possible for this to just exercise one instance.